common/changes/office-ui-fabric-react/sayalis-makeSearchHasFocusPublic_2018-03-19-20-51.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Add `hasFocus` public api in `SearchBox` that returns a flag indicating whether `SearchBox` has focus or not.",
+      "type": "patch"
```

Comment:
Would argue this should be a minor change since you're adding functionality.

Reply:
ok

Reply:
You have to manually delete this file still.

Reply:
Done, thanks